homeworks/denys.viitenko_danviitenko/homework_6/MA-github-client/MA-github-client/Search/SearchViewController.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  SearchViewController.swift
+//  MA-github-client
+//
+//  Created by Dan on 12/17/18.
+//  Copyright © 2018 Dan Viitenko. All rights reserved.
+//
+
+import UIKit
+
+class SearchViewController: UIViewController, UITableViewDelegate,UITableViewDataSource,UISearchBarDelegate {
+    var repositories = [Repo]()
+    @IBOutlet weak var searchBar: UISearchBar!
+    @IBOutlet weak var tblView: UITableView!
+    
+    var dataFiltered: [Repo]!
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        addRepo()
+        let nib = UINib.init(nibName: "RepoTableViewCell", bundle: nil)
```

Comment:
В качестве имени лучше всего использовать константы.

```suggestion
        let nib = UINib(nibName: "RepoTableViewCell", bundle: nil)
```